src/IterableCodeExtractor.php

```diff
@@ -232,6 +233,11 @@ static function ( $file, $key, $iterator ) use ( $include, $exclude, $extensions
 						return static::containsMatchingChildren( $file, $include );
 					}
 
+					// Include directories that are excluded but include score is higher.
+					if ( 0 < $exclusion_score && $inclusion_score >= $exclusion_score && $iterator->hasChildren() ) {
```

Comment:
Nit: no yoda conditions here.

```suggestion
					if ( $exclusion_score > 0 && $inclusion_score >= $exclusion_score && $iterator->hasChildren() ) {
```

Reply:
Addressed in b079a92.